docs/dev/pos/gui.rst

```diff
@@ -0,0 +1,37 @@
+JS access and inheritance
+=========================
+
+action_button
+-------------
+
+Here you will find explanation of how to get/inherit action_button POS objects.
+
+For example we got JS object::
+
+    odoo.define('pos_reprint.pos_reprint', function (require) {
+    ...
+    screens.define_action_button({
+        'name': 'reprint',
+        'widget': ReprintButton,
+        'condition': function()
+
+Owner object (pos_reprint.pos_reprint) do not exports (returns) any data.
```

Comment:
This defenition doesn't return class ReprintButton. So, we cannot inherit it in a usual way